src/vanta_ledger/routes/companies.py

```
@@ -0,0 +1,116 @@
+            "limit": limit,
+            "total_pages": (total_count + limit - 1) // limit
+        }
+    finally:
+        cursor.close()
+        conn.close()
+
+@router.get("/")
+async def get_companies(
+    page: int = 1,
+    limit: int = 20,
+    current_user: dict = Depends(AuthService.verify_token)
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)